pkg/api/store/cluster/cluster_store.go

```diff
@@ -17,3 +25,67 @@ func (r *Store) ByID(apiContext *types.APIContext, schema *types.Schema, id stri
 	}
 	return r.Store.ByID(apiContext, schema, id)
 }
+
+func (r *Store) Create(apiContext *types.APIContext, schema *types.Schema, data map[string]interface{}) (map[string]interface{}, error) {
+
+	name, ok := data["name"].(string)
```

Comment:
we should extract lines 31-34 into common method

Reply:
done